### PR TITLE
Russian translation: Get started

### PR DESCRIPTION
**Pull request by DreaMinder**
Ever since version 1.*, the Russian version of the documentation has been outdated. This is the first part of updates and follow-up support.

This part includes
- i18n file translation
- guides/get-started section translation

The translation is based on the [previous](https://github.com/nuxt/docs/pull/2116) translation-attempt.

### Kudos to
@cogor
@icehaunter
@sp1r1don
@merzavko
@Domovikx
@Alex-Sokolov
@MaximStone
@theartkod

### CODE REVIEW

**Comment by DreaMinder** [comment]
hey, @aldarund, wanna review? 😉

File: content/ru/guides/get-started/directory-structure.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Структура директорий
+description: Стандартная структура Nuxt.js приложения призвана стать хорошей отправной точкой для больших и малых проектов. Тем не менее, вы можете создать нужную вам структуру директорий.
+position: 3
+category: get-started
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/01_get_started/03_directory_structure?fontsize=14&hidenavigation=1&theme=dark
+---
+
+Стандартная структура Nuxt.js приложения призвана стать хорошей отправной точкой для больших и малых проектов. Тем не менее, вы можете создать нужную вам структуру директорий.
```
Comment (DreaMinder) [inline content/ru/guides/get-started/directory-structure.md:9]:
```suggestion
Стандартная структура Nuxt.js приложения призвана стать хорошей отправной точкой для больших и малых проектов. Тем не менее, вы можете организовать собственную структуру директорий.
```

File: content/ru/guides/get-started/directory-structure.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Файл `package.json` включает зависимости и скрипты вашего приложения.
+
+<app-modal>
+  <code-sandbox  :src="csb_link"></code-sandbox>
+</app-modal>
+
+## Больше о структуре проекта
+
+Существуют еще полезные директории , включая [content](/guides/directory-structure/content), [layouts](/guides/directory-structure/layouts), [middleware](/guides/directory-structure/middleware), [modules](/guides/directory-structure/modules), [plugins](/guides/directory-structure/plugins) и [store](/guides/directory-structure/store). Они не нужны для небольших приложений и здесь мы их не рассматриваем.
```
Comment (DreaMinder) [inline content/ru/guides/get-started/directory-structure.md:91]:
```suggestion
Существуют другие полезные директории: [content](/guides/directory-structure/content), [layouts](/guides/directory-structure/layouts), [middleware](/guides/directory-structure/middleware), [modules](/guides/directory-structure/modules), [plugins](/guides/directory-structure/plugins) и [store](/guides/directory-structure/store). Они не нужны для небольших приложений и здесь мы их не рассматриваем.
```

File: content/ru/guides/get-started/routing.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+<base-alert type="next">
+
+Подробнее о [Маршрутизации](/guides/features/file-system-routing)
+
+</base-alert>
+
+## Навигация
+
+Доя навигации между страницами вашего приложения вы должны использовать компонент [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component). Этот компонент - часть Nuxt.js и, следовательно, его не нужно импортировать, как вы обычно это делаете с другими компонентами. Он похож на HTML тег `<a>`, за исключением того, что вместо `href="/about"` мы используем `to="/about"`. Если ранее вы использовали `vue-router`, то можете считать `<NuxtLink>` заменой `<RouterLink>`
```
Comment (DreaMinder) [inline content/ru/guides/get-started/routing.md:23]:
```suggestion
Для навигации между страницами вашего приложения вы должны использовать компонент [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component). Этот компонент - часть Nuxt.js и, следовательно, его не нужно импортировать, как вы обычно это делаете с другими компонентами. Он похож на HTML тег `<a>`, за исключением того, что вместо `href="/about"` мы используем `to="/about"`. Если ранее вы использовали `vue-router`, то можете считать `<NuxtLink>` заменой `<RouterLink>`
```

File: content/ru/guides/get-started/upgrading.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static'
+}
+```
+
+## Getting Started
+
+1. Для версии, которую вы хотите - проверьте [release notes](/guide/release-notes) чтобы узнать, есть ли какие-либо дополнительные инструкции для этого конкретного релиза.
```
Comment (DreaMinder) [inline content/ru/guides/get-started/upgrading.md:20]:
```suggestion
1. Проверьте [release notes](/guide/release-notes) для желаемой версии чтобы узнать, есть ли какие-либо дополнительные инструкции для этого конкретного релиза.
```

**Review by DreaMinder — COMMENTED** [review]
Added a couple of fixes

File: content/ru/guides/get-started/conclusion.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      - http://localhost:3000/project-name:3000
+      - http://localhost:3000/nuxt:3000/
+    correctAnswer: http://localhost:3000/
+  - question: В каком файле можно изменять конфигурацию Nuxt.js?
+    answers:
+      - nuxt.config.json
+      - config.js
+      - nuxt.config.js
+    correctAnswer: nuxt.config.js
+  - question: Какая дериктория не подходит для размещения файлов с расширением `.vue`?
```
Comment (taraskhvyl) [inline content/ru/guides/get-started/conclusion.md:37]: "директория"

File: content/ru/guides/get-started/conclusion.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    answers:
+      - pages
+      - static
+      - components
+    correctAnswer: static
+  - question: В какой дериктории Вы будете размещать файлы с Вашими стилями?
```
Comment (taraskhvyl) [inline content/ru/guides/get-started/conclusion.md:43]: "директории"

File: content/ru/guides/get-started/conclusion.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    answers:
+      - styles
+      - components
+      - assets
+    correctAnswer: assets
+  - question: В какой дериктории размещается robots.txt или favicon?
```
Comment (taraskhvyl) [inline content/ru/guides/get-started/conclusion.md:49]: "директории"

File: content/ru/guides/get-started/conclusion.md (hunk 5 above)
Comment (DreaMinder) [inline content/ru/guides/get-started/conclusion.md:37]:
```suggestion
  - question: Какая директория не подходит для размещения файлов с расширением `.vue`?
```

File: content/ru/guides/get-started/conclusion.md (hunk 6 above)
Comment (DreaMinder) [inline content/ru/guides/get-started/conclusion.md:43]:
```suggestion
  - question: В какой директории вы будете размещать файлы с Вашими стилями?
```

File: content/ru/guides/get-started/conclusion.md (hunk 7 above)
Comment (DreaMinder) [inline content/ru/guides/get-started/conclusion.md:49]:
```suggestion
  - question: В какой директории размещается robots.txt или favicon?
```

File: content/ru/guides/get-started/conclusion.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+      - '<NuxtLink>'
+    correctAnswer: '<NuxtLink>'
+  - question: '`<NuxtLink>` используется для внутренних ссылок, которые принадлежат приложению Nuxt.js?'
+    answers:
+      - Да
+      - Нет
+    correctAnswer: Да
+---
+
+Поздравляем! Вы создали Ваше первое приложение на Nuxt.js и теперь можете смело называть себя Nuxter. Но Вам еще предстоит многому научиться чтобы увидеть все возможности Nuxt.js. У нас есть несколько рекомендаций для Вашего последующего обучения:
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/conclusion.md:68]:
```suggestion
Поздравляем! Вы создали Ваше первое приложение на Nuxt.js и теперь можете смело называть себя Nuxter. Но Вам еще предстоит многому научиться, чтобы увидеть все возможности Nuxt.js. У нас есть несколько рекомендаций для Вашего последующего обучения:
```

File: content/ru/guides/get-started/conclusion.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+<base-alert type="next">
+
+Ознакомтесь с [Concepts book](/guides/concepts/views)
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/conclusion.md:72]:
```suggestion
Ознакомьтесь с [Concepts book](/guides/concepts/views)
```

**Review by DreaMinder — COMMENTED** [review]
A couple of final fixes based on @taraskhvyl suggestions

File: content/ru/guides/get-started/directory-structure.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+[Подробнее о страницах](/guides/directory-structure/pages)
+
+</base-alert>
+
+### Директория `components`
+
+
+Директория `components` включает ваши Vue.js компоненты, которые вы будете импортировать в компоненты страниц.
+
+С Nuxt.js вы можете создать ваши компоненты и автоматически их импортировать в ваши .vue файлы не указывая при этом явно импорт этих компонентов в секции script. Nuxt.js будет сканировать и импортировать ваши компоненты автоматически, если в файле конфигурации установлено `components: true`.
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/directory-structure.md:43]:
```suggestion
С Nuxt.js вы можете создать ваши компоненты и автоматически их импортировать в ваши `.vue` файлы, не указывая при этом явно импорт этих компонентов в секции script. Nuxt.js будет сканировать и импортировать ваши компоненты автоматически, если в файле конфигурации установлено `components: true`.
```

File: content/ru/guides/get-started/routing.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+title: Маршрутизация
+description: Большинство сайтов имеют больше одной страницы. К примеру: главная страница, страница о нас, страница контактов и т.д. Для показа этих страниц нам понадобится Маршрутизатор.
+position: 2
+category: get-started
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/01_get_started/02_routing?fontsize=14&hidenavigation=1&theme=dark
+---
+
+## Автоматическая маршрутизация
+
+Большинство сайтов имеют более одной страницы (К примеру: главная страница, страница о нас, страница контактов и т.д.). Для отображения этих страниц нам понадобится Маршрутизатор. С этим нам поможет `vue-router`. При работе с Vue.js приложением у нас был файл конфигурации (`router.js`), в который мы добавляли свои маршруты. Nuxt.js автоматически генерирует конфигурационный файл `vue-router` на основе `.vue` файлов  внутри директории `pages`. Поэтому вам не придется писать конфигурацию роутера самостоятельно! Nuxt.js также предоставляет автоматическое разделение кода (code-splitting) для всех маршрутов.
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/routing.md:11]:
```suggestion
Большинство сайтов имеют более одной страницы (например: главная страница, страница о нас, страница контактов и т.д.). Для отображения этих страниц нам понадобится маршрутизатор. С этим нам поможет `vue-router`. При работе с Vue.js приложением у нас был файл конфигурации (`router.js`), в который мы добавляли свои маршруты. Nuxt.js автоматически генерирует конфигурационный файл `vue-router` на основе `.vue` файлов  внутри директории `pages`. Поэтому вам не придется писать конфигурацию роутера самостоятельно! Nuxt.js также предоставляет автоматическое разделение кода (code-splitting) для всех маршрутов.
```

File: content/ru/guides/get-started/routing.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+
+<base-alert type="next">
+
+Подробнее о [Маршрутизации](/guides/features/file-system-routing)
+
+</base-alert>
+
+## Навигация
+
+Для навигации между страницами вашего приложения вы должны использовать компонент [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component). Этот компонент - часть Nuxt.js и, следовательно, его не нужно импортировать, как вы обычно это делаете с другими компонентами. Он похож на HTML тег `<a>`, за исключением того, что вместо `href="/about"` мы используем `to="/about"`. Если ранее вы использовали `vue-router`, то можете считать `<NuxtLink>` заменой `<RouterLink>`
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/routing.md:23]:
```suggestion
Для навигации между страницами вашего приложения вы должны использовать компонент [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component). Этот компонент - часть Nuxt.js и, следовательно, его не нужно импортировать, как вы обычно это делаете с другими компонентами. Он похож на HTML-тег `<a>`, за исключением того, что вместо `href="/about"` мы используем `to="/about"`. Если ранее вы использовали `vue-router`, то можете считать `<NuxtLink>` заменой `<RouterLink>`
```

File: content/ru/guides/get-started/upgrading.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
+---
+title: Обновление
+description: Обновление Nuxt.js выполняется быстро, но требует больше усилий, чем обновление вашего package.json
+position: 5
+category: get-started
+---
+
+> Обновление Nuxt.js выполняется быстро, но требует больше усилий, чем обновление вашего package.json
+
+Если вы хотите использовать статический хостинг при обновлении Nuxt v2.14 - вам понадобится добавить [target:static](/guides/features/deployment-targets#static-hosting) в ваш nuxt.config.js, для корректной работы команды `generate`.
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/upgrading.md:10]:
```suggestion
Если вы хотите использовать статический хостинг при обновлении Nuxt v2.14 - вам понадобится добавить [target:static](/guides/features/deployment-targets#static-hosting) в ваш nuxt.config.js для корректной работы команды `generate`.
```

File: content/ru/guides/get-started/upgrading.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static'
+}
+```
+
+## Getting Started
+
+1. Проверьте [release notes](/guide/release-notes) для желаемой версии чтобы узнать, есть ли какие-либо дополнительные инструкции для этого конкретного релиза.
```
Comment (NataliaTepluhina) [inline content/ru/guides/get-started/upgrading.md:20]:
```suggestion
1. Проверьте [release notes](/guide/release-notes) для желаемой версии, чтобы узнать, есть ли какие-либо дополнительные инструкции для этого конкретного релиза.
```

File: content/ru/guides/get-started/directory-structure.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+[Подробнее о страницах](/guides/directory-structure/pages)
+
+</base-alert>
+
+### Директория `components`
+
+
+Директория `components` включает ваши Vue.js компоненты, которые вы будете импортировать в компоненты страниц.
+
+С Nuxt.js вы можете создать ваши компоненты и автоматически их импортировать в ваши `.vue` файлы, не указывая при этом явно импорт этих компонентов в секции script. Nuxt.js будет сканировать и импортировать ваши компоненты автоматически, если в файле конфигурации установлено `components: true`.
```
Comment (DreaMinder) [inline content/ru/guides/get-started/directory-structure.md:43]:
```suggestion
С Nuxt.js вы можете создать компоненты и автоматически импортировать их в любые `.vue` файлы, не указывая при этом явно импорт этих компонентов в секции script. Nuxt.js будет сканировать и импортировать ваши компоненты автоматически, если в файле конфигурации установлено `components: true`.
```

File: content/ru/guides/get-started/installation.md [hunk 16]
```diff
@@ -0,0 +1,244 @@
...
+
+<base-alert type="info">
+
+Мы используем команду dev когда нам нужно запустить приложение в режиме разработки.
+
+</base-alert>
+
+Теперь приложение запущено по адресу **[http://localhost:3000](http://localhost:3000/).**
+
+Откройте ваше приложение нажав на ссылку в терминале и вы увидите текст "Привет мир!", который мы добавили на предыдущем шаге.
```
Comment (DreaMinder) [inline content/ru/guides/get-started/installation.md:160]:
```suggestion
Откройте ваше приложение, нажав на ссылку в терминале, и вы увидите текст "Привет мир!", который мы добавили на предыдущем шаге.
```

File: content/ru/guides/get-started/installation.md [hunk 17]
```diff
@@ -0,0 +1,244 @@
...
+    "start": "nuxt start"
+  }
+}
+```
+
+В секции `scripts` указаны Nuxt.js команды, которые будут запущены при выполнении `npm run <имя_команды>`.
+
+#### **Что такое файл package.json?**
+
+`package.json` это как паспорт вашего проекта. Если вы хотите больше узнать о файле `package.json`, то мы рекомендуем почитать [документацию NPM](https://docs.npmjs.com/creating-a-package-json-file).
```
Comment (DreaMinder) [inline content/ru/guides/get-started/installation.md:74]:
```suggestion
`package.json` - это как паспорт вашего проекта. Если вы хотите больше узнать о файле `package.json`, то мы рекомендуем почитать [документацию NPM](https://docs.npmjs.com/creating-a-package-json-file).
```

**Comment by DreaMinder** [comment]
@debs-obrien Looks like this PR is ready to be merged 🥳

Thanks for the review, @taraskhvyl, and @NataliaTepluhina!

File: i18n/ru-RU.js [hunk 18]
```diff
@@ -125,239 +125,239 @@ module.exports = {
...
-    msg1: 'Want to take a look at our new docs? Our ',
-    link: 'new docs',
-    msg2: 'are now in beta. Have fun!'
+    msg1: 'Хотите взглянуть на нашу новую документацию? Наша ',
+    link: 'новая документация',
+    msg2: 'в бета-версии. Развлекайтесь!'
   },
   contribute: {
-    title: 'Contributors',
+    title: 'Помощники',
```
Comment (DreaMinder) [inline i18n/ru-RU.js:294]:
```suggestion
    title: 'Авторы',
```

File: content/ru/guides/get-started/conclusion.md [hunk 19]
```diff
@@ -0,0 +1,94 @@
+---
+title: Заключение
+description: Поздравляем! Вы создали Ваше первое приложение на Nuxt.js и теперь можете смело называть себя Nuxter. Но Вам еще предстоит многому научиться чтобы увидеть все возможности Nuxt.js. У нас есть несколько рекомендаций для Вашего последующего обучения.
```
Comment (DreaMinder) [inline content/ru/guides/get-started/conclusion.md:3]:
```suggestion
description: Поздравляем! Вы создали ваше первое приложение на Nuxt.js и теперь можете смело называть себя Nuxter. Но Вам еще предстоит многому научиться чтобы увидеть все возможности Nuxt.js. У нас есть несколько рекомендаций для последующего обучения.
```

**Comment by DreaMinder** [comment]
Hey @debs-obrien, is there anything we could do to get this merged?

**Comment by debs-obrien** [comment]
Thanks @DreaMinder and all involved. great work 💪

**Pull request merged**